packages/fast-components-styles-msft/src/pivot/index.ts

```diff
@@ -63,12 +63,14 @@ const styles: ComponentStyles<PivotClassNameContract, DesignSystem> = (
         pivot_tab__active: {},
         pivot_tabContent: {
             transition: "all 0.2s ease-in-out",
-            ...applyTypeRampConfig("t7"),
+            ...scaleApplyTypeRampConfigWithDensity(designSystem, "t7"),
+            position: "relative",
+            top: "-2px",
         },
         pivot_activeIndicator: {
             position: "absolute",
-            borderRadius: toPx(designSystem.cornerRadius),
-            top: density(27),
+            ...applyCornerRadius(designSystem),
+            top: toPx(heightNumber(1)(designSystem) - 3 - 2) /* height & padding */,

```

Comment:
are 3 and 2 significant here and representative of something? I'd recommend either setting as a const so it's clear, or just writing "5".

Reply:
Updated to use declarative constants.